feat(bitgo): add BitGo Korea entity

pranishnepal opened the pull request:
## What
Currently, "BiGo Sister Trust 1" represents BitGo Korea entity. This commit adds "BitGo Korea" as a new entity since that would be safer than changing the existing.

## Issue Number
TICKET: COPS-2076

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update


## Testing
- Unit Tests 
- `yarn run unit-test --scope @bitgo/statics`

<!--
# Please be aware of the following when making your pull request:

## Description

Please include a summary of your proposed changes and which issue is being addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Issue Number

Internal Users - Please include the related internal tracking number (e.g. BG-000000).
External Users - Please link to any relevant github issues as necessary.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code compiles correctly for both Node and Browser environments
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My commits follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) and I have properly described any BREAKING CHANGES
- [ ] The ticket or github issue was included in the commit message as a reference
- [ ] I have made corresponding changes to the documentation and on any new/updated functions and/or methods - [jsdoc](https://jsdoc.app/)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
-->

aarvay commented on modules/statics/src/base.ts:
```diff
@@ -213,9 +213,14 @@ export enum CoinFeature {
   /*
-   * This coin supports custody in BitGo Sister Trust 1 entities
+   * This coin supports custody in BitGo Sister Trust 1 entities.
+   * This used to represent BitGo Korea previously. Going forward, use BitGo Korea instead.
```
```suggestion
   * @deprecated This used to represent BitGo Korea previously. Going forward, use `CUSTODY_BITGO_KOREA` instead.
```

pranishnepal replied:
updated, thanks!

abhishekdhakla commented on yarn.lock:
I think we don't need this one!

pranishnepal replied:
i thought about it, but didn't see it in `.gitignore` so i let it commit. Shall i add it to `.gitignore,` then?

pranishnepal replied:
removed + added to `.gitignore`

pranishnepal committed:
feat(bitgo): add BitGo Korea entity

Currently, "BiGo Sister Trust 1" represents BitGo Korea entity.
This commit adds "BitGo Korea" entity as adding a new entity would be
safer rather than changing the existing.

TICKET: COPS-2076
